### ISSUE TITLE
Feature Request Function to generate 100 or more images

### ISSUE THREAD

**Issue by iamapake** [divamgupta/diffusionbee-stable-diffusion-ui#289]
I would like to use a Function to generate 100 or more images and  a function to generate images forever.
thank you.

**Comment by Gitterman69** [comment]
yes please!!!! also higher resolution please - max 1024 or 1500 or something would be great to make ultra wide images

**Comment by impallari** [comment]
1024px please!!!
